src/java/org/apache/cassandra/net/MessagingService.java

```diff
@@ -665,9 +665,9 @@ public void updateBackPressureOnSend(InetAddressAndPort host, IAsyncCallback cal
     /**
      * Updates the back-pressure state on reception from the given host if enabled and the given message callback supports it.
      *
-     * @param host The replica host the back-pressure state refers to.
+     * @param host     The replica host the back-pressure state refers to.
```

Comment:
Your IDE is set up to fix whitespace on unchanged lines. That's generally not great just because it introduces a lot of extra changes, and if two people have IDEs with different preferences they end up fighting and introducing random conflicts all the time.

Can you go through this change set and remove whitespace changes for things that haven't actually been touched?

Reply:
Interesting, I'm just using the default style guide for idea that came with Cassandra (`ant generate-idea-files`). If it's just that review is tough you can tell github not to show you whitespace changes.

That's good feedback though that you don't want unrelated whitespace changes, I'll take it out :-)

Reply:
If you go to IntelliJ preferences, search for saved, under Editor -> General -> Strip trailing spaces you can select modified lines from the drop down. Not sure if that impacts this particular reformatting here.

Reply:
Ack, all setup thanks :-)